editors/code/src/commands/syntax_tree.ts

```diff
@@ -22,6 +22,7 @@ export function syntaxTree(ctx: Ctx): Cmd {
             if (doc.languageId !== 'rust') return;
             afterLs(() => tdcp.eventEmitter.fire(tdcp.uri));
         },
+        null,
```

Comment:
Oh wow. Type safety at its finest.

:100: :+1:

Reply:
I never loved dynamic typing, I actually don't know how I ended up in JavaScript world :cry: